src/main/java/it/pagopa/pn/deliverypush/action/refinement/RefinementHandler.java

```diff
@@ -12,19 +13,24 @@
 @Slf4j
 @Component
 public class RefinementHandler {
+    private static final int RETENTION_FILE_REFINEMENT_DAYS = 120;
```

Comment:
Non è duplicato rispetto a quello in src/main/java/it/pagopa/pn/deliverypush/action/notificationview/ViewNotification.java ?

Reply:
Sì, volendo possiamo creare una classe Constant che contiene questa costante così che venga richiamata sia da ViewNotification sia da RefinementHandler.

Reply:
RetentionConstants, ma normalmente tutti i parametri sono estrenalizzati nella configurazione in modo che si possa cambiare da ambiente ad un altro.